cluster/olm/ceph/generate-rook-csv.sh

```diff
@@ -222,26 +222,26 @@ function hack_csv() {
     # rook-ceph-osd --> serviceAccountName
     #     rook-ceph-osd --> rule
 
-    sed -i 's/rook-ceph-global/rook-ceph-system/' "$CSV_FILE_NAME"
-    sed -i 's/rook-ceph-object-bucket/rook-ceph-system/' "$CSV_FILE_NAME"
-    sed -i 's/rook-ceph-cluster-mgmt/rook-ceph-system/' "$CSV_FILE_NAME"
+    sed -i'' -e 's/rook-ceph-global/rook-ceph-system/' "$CSV_FILE_NAME"
```

Comment:
Can we put `sed -i'' -e` in an array variable and then call it instead? Who knows when this line might change again...

Reply:
or export `SED_CMD` from Make?